File: terminals/cs/cancel-terminal-session.cs
var client = new RestClient("https://api.dojo.tech/terminal-sessions/%7BterminalSessionId%7D/cancel");
var request = new RestRequest(Method.PUT);
request.AddHeader("version", "SOME_STRING_VALUE");
request.AddHeader("Authorization", "REPLACE_KEY_VALUE");
IRestResponse response = client.Execute(request);

File: terminals/python/cancel-terminal-session.py
import requests

url = "https://api.dojo.tech/terminal-sessions/%7BterminalSessionId%7D/cancel"

headers = {
    "version": "SOME_STRING_VALUE",
    "Authorization": "REPLACE_KEY_VALUE"
}

response = requests.request("PUT", url, headers=headers)

print(response.text)